css.js

```diff
@@ -53,16 +66,25 @@ if (typeof window !== 'undefined') {
       link.onerror = function(event) {
         _callback(event.error || new Error('Error loading CSS file.'));
       };
-      head.appendChild(link);
+      head.insertBefore(link, existingLinks[0]);
+    })
+    .then(function(result){ 
+      forEach(existingLinks, function(link){link.parentElement.removeChild(link);})
+      return result;
+    })
+    .catch(function(err){
+      forEach(existingLinks, function(link){link.parentElement.removeChild(link);})
```

Comment:
Can you explain why this duplication is needed? Would it not be better to catch on individual `removeChild` calls?